### PR TITLE
[New Package] telepathy_logger

### PR DESCRIPTION
**Pull request by saltedcoffii**

### CODE REVIEW

File: packages/telepathy_logger.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  compatibility 'all'
+  source_url 'https://telepathy.freedesktop.org/releases/telepathy-logger/telepathy-logger-0.8.2.tar.bz2'
+  source_sha256 '8fcad534d653b1b365132c5b158adae947810ffbae9843f72dd1797966415dae'
+
+  depends_on 'dbus_glib'
+  depends_on 'libx11'
+  depends_on 'gobject_introspection'
+  depends_on 'telepathy_glib'
+  depends_on 'sqlite'
+  depends_on 'python27' => :build
```
Comment (uberhacker) [inline packages/telepathy_logger.rb:16]: `sqlite` and `python27` are core packages?

Comment (saltedcoffii) [reply packages/telepathy_logger.rb:16]: SQLite depends on most, i'll remove it though. Python2.7 is depreciated and will likely eventually need to be removed from the core package list. It wouldn't hurt to start listing it in the dependecy lists of packages.

Comment (uberhacker) [reply packages/telepathy_logger.rb:15]: Python 2 is not going away anytime soon so it should be fine to remove.

**Comment by satmandu** [comment]
Here's a modification of this package I was working on a couple of weeks ago adapted from arch which resolves the python3 issue:
```
require 'package'

class Telepathy_logger < Package
  description 'Telepathy framework logging daemon'
  homepage 'http://telepathy.freedesktop.org/wiki/Logger'
  @_ver = '0.8.2'
  version @_ver
  compatibility 'all'
  source_url "https://telepathy.freedesktop.org/releases/telepathy-logger/telepathy-logger-#{@_ver}.tar.bz2"
  source_sha256 '8fcad534d653b1b365132c5b158adae947810ffbae9843f72dd1797966415dae'

  depends_on 'telepathy_glib'
  depends_on 'dconf'
  depends_on 'gobject_introspection' => ':build'
  depends_on 'libxslt' => ':build'

  def self.patch
    patch_description = 'Fix build with Python 3'
    patch_url = 'https://github.com/archlinux/svntogit-packages/raw/packages/telepathy-logger/trunk/0001-tools-Fix-the-build-with-Python-3.patch'
    patch_sha256 = 'd94899bed85a4ac90158838524d16c761956eec2cd0d223b6a18bffa09ec008b'
    patch_filename = 'patch'
    puts "Downloading patch: #{patch_description}".yellow
    system('curl', '-s', '--insecure', '-L', '-#', patch_url, '-o', patch_filename)
    abort 'Checksum mismatch. :/ Try again.'.lightred unless
    Digest::SHA256.hexdigest(File.read(patch_filename)) == patch_sha256
    puts 'patch downloaded'.lightgreen
    system 'patch -p 1 < patch'

    system "sed -i -r 's:\"(/system):\"/org/freedesktop\1:g' data/*.in"
    system "sed -i -r 's:\"(/apps):\"/org/freedesktop\1:g' data/*.in"
  end

  def self.build
    system  "env CFLAGS='-pipe -fno-stack-protector -U_FORTIFY_SOURCE -flto=auto' \
      CXXFLAGS='-pipe -fno-stack-protector -U_FORTIFY_SOURCE -flto=auto' \
      LDFLAGS='-fno-stack-protector -U_FORTIFY_SOURCE -flto=auto' \
      ./configure \
      #{CREW_OPTIONS} \
      --sysconfdir=#{CREW_PREFIX}/etc \
      --libexecdir=#{CREW_LIB_PREFIX}/telepathy \
      --enable-call \
      --disable-scrollkeeper \
      --disable-gtk-doc \
      --enable-introspection \
      --enable-public-extensions \
      --with-x"
    system 'make'
  end

  def self.install
    system "make DESTDIR=#{CREW_DEST_DIR} install"
  end

  def self.postinstall
    system "glib-compile-schemas #{CREW_PREFIX}/share/glib-2.0/schemas/"
  end
end
```

**Comment by satmandu** [comment]
Actually, let me also upload the telepathy_glib package with binaries I built then too.

File: packages/telepathy_logger.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  source_url 'https://telepathy.freedesktop.org/releases/telepathy-logger/telepathy-logger-0.8.2.tar.bz2'
+  source_sha256 '8fcad534d653b1b365132c5b158adae947810ffbae9843f72dd1797966415dae'
+
+  depends_on 'dbus_glib'
+  depends_on 'libx11'
+  depends_on 'gobject_introspection'
+  depends_on 'telepathy_glib'
+  
+  def self.build
+    system "env PYTHON=python2.7 \
```
Comment (uberhacker) [inline packages/telepathy_logger.rb:17]: Can we use python 3?

Comment (satmandu) [reply packages/telepathy_logger.rb:17]: package file attached above uses python3 and is patched for such.

Comment (satmandu) [reply packages/telepathy_logger.rb:17]: Actually, I take it back. telepathy logger is both broken and abandoned, as far as I can tell, and I can't get it to compile properly.

Comment (saltedcoffii) [reply packages/telepathy_logger.rb:17]: that's what I was noticing, maybe there are some patches for it on salsa.debian.org. there usually are.

**Comment by uberhacker** [comment]
@saltedcoffii: Is there any more progress on this PR?

**Comment by saltedcoffii** [comment]
no :(

**Pull request closed without merging**